applications/API/serializers.py

```
@@ -0,0 +1,569 @@
+        with contextlib.suppress(Exception):
+            from applications.API.models import (
+                InsultCategory as _IC,  # local import to avoid cycles
+            )
+
+            if isinstance(value, _IC):
+                return value.category_key
+        if not isinstance(value, str):
+            return value
+        v = value.strip()
+        # Try common separators first
+        for sep in (" - ", "–", "-"):
```

Ruff (0.12.2)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)